app/javascript/react/components/Graph/chartEvents.ts

```diff
@@ -58,22 +66,53 @@ const addNavigationArrows = (chart: Highcharts.Chart) => {
   const createArrows = () => {
     const chartWidth = chart.chartWidth;
     const chartHeight = chart.chartHeight;
-    const chevronHeight =
-      window.innerWidth < 1025 ? chartHeight / 2 : chartHeight / 2 - 24;
 
     // Remove existing arrows if any
     chart.renderer.boxWrapper.element
       .querySelectorAll(".custom-arrow")
       .forEach((el) => el.remove());
 
+    const leftIcon = isMobile
+      ? mobileChevronLeft
+      : isCalendarPage
+      ? chevronLeft
+      : graphChevronLeft;
+    const rightIcon = isMobile
+      ? mobileChevronRight
+      : isCalendarPage
+      ? chevronRight
+      : graphChevronRight;
+
+    let iconSize = 48;
+    let leftArrowX = 15;
+    let rightArrowX = chartWidth - 118;
+    let arrowY = chartHeight / 2;
+
+    if (isMobile && isCalendarPage) {
+      iconSize = 40;
+      leftArrowX = 0;
+      rightArrowX = chartWidth - 50;
+      arrowY = -29;
+    } else if (!isMobile && isCalendarPage) {
+      iconSize = 46;
+      leftArrowX = -70;
+      rightArrowX = chartWidth + 25;
+      arrowY = chartHeight / 2 - 24;
+    } else if (!isMobile && !isCalendarPage) {
+      iconSize = 48;
+      leftArrowX = 15;
+      rightArrowX = chartWidth - 118;
+      arrowY = chartHeight / 2 - 24;
+    }
```

Comment:
if someone asks me why everything takes so long, I'd point them in this direction - the amount of conditions to place the arrows as in designs can take some time to figure out, implement or fix